src/main/java/core/basesyntax/Excalator.java

```diff
@@ -0,0 +1,4 @@
+package core.basesyntax;
+
+public class Excalator extends Machine {
+}

```

Comment:
The `doWork()` and `stopWork()` methods are missing in the `Excalator` (should be `Excavator`) class. Override these methods to print messages as mentioned in the task description. For example:

@Override
public void doWork() {
  System.out.println("Excavator is working");
}

@Override
public void stopWork() {
  System.out.println("Excavator has stopped working");
}
🚧